it/mail/chrome/messenger/comm4xMailImportMsgs.properties

```diff
@@ -58,10 +58,10 @@
 # Error Message
 ## @name COMM4XMAILIMPORT_MAILBOX_BADPARAM
 ## @loc None
-2003=Si è verificato un errore interno, importazione non riuscita. Tentare nuovamente l'importazione.
+2003=Si è verificato un errore interno, importazione non riuscita. Riprova a importare.
```

Comment:
CORRECTIONS NEEDED. 

In order to keep the consistency, please use the same sentence used in another file (that is, replace "Riprova a importare" with "Prova ad importare di nuovo")